sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -136,6 +140,156 @@ _rom_ext_start_boot:
   li   t0, 0xFFFF0888
   csrc mie, t0
 
+  /**
+   * Initialize the Ibex simple address translation to run position dependent
+   * code in the virtual flash space.
+   * 
+   * Before the setup of the address translation the assembly code shall be 
+   * position independent, however we don't want to use the `.option pic` 
+   * directive as it requires a `got` (Global Offset Table) section. 
+   * Therefore pseudoinstructions such as `la` must be avoided because it is
+   * expanded differently for`pic` e `nopic` code.
+   * i.e:
+   *   `la rd, symbol` expands to: `pic`   -> `auipc rd, offsetHi`
+   *                                          `lw rd, offsetLo(rd)`
+   *
+   *                               `nopic` -> `auipc rd, offsetHi`
+   *                                          `addi rd, offsetLo`
+   *
+   * The `pic` expansion will use the got whilst the `nopic` expansion will
+   * assume the linked address to pc which will be wrong when executing in a
+   * different address space, resulting in a wrong value loaded into `rd`.
+   * 
+   * The alternative used is to replace:
```

Comment:
Maybe add something about this deliberately loading the absolute address to emphasise what is going on?